packages/@lwc/perf-benchmarks/package.json

```diff
@@ -26,7 +26,8 @@
         "@best/cli": "^8.1.3",
         "@best/runner-headless": "^8.1.3",
         "@best/runner-remote": "^8.1.3",
-        "@lwc/rollup-plugin-node-resolve-v13": "npm:@rollup/plugin-node-resolve@13.0.5",
+        "@lwc/rollup-plugin-node-resolve-legacy": "npm:rollup-plugin-node-resolve@5.2.0",
+        "@lwc/rollup-plugin-commonjs-legacy": "npm:rollup-plugin-commonjs@10.1.0",
```

Comment:
It turns out we need super-old versions of these plugins, so that they can be compatible both with Best's Rollup version (~1.26.0) and with each other. These are from 4 years ago.